tests/wasmfs/wasmfs_chdir.c

```diff
@@ -22,58 +22,58 @@ int main() {
 
   // Try to print the root directory.
   char cwd[100];
-  getcwd(cwd, sizeof(cwd));
+  printf("getcwd() result: %s\n", getcwd(cwd, sizeof(cwd)));
   printf("Current working dir: %s\n", cwd);
 
   // Try to switch to /working.
   chdir("working");
-  getcwd(cwd, sizeof(cwd));
+  printf("getcwd() result: %s\n", getcwd(cwd, sizeof(cwd)));
   printf("Current working dir: %s\n", cwd);
 
   // Try to switch back to the root directory.
   chdir("/");
-  getcwd(cwd, sizeof(cwd));
+  printf("getcwd() result: %s\n", getcwd(cwd, sizeof(cwd)));
   printf("Current working dir: %s\n", cwd);
 
   // Try to switch to a subdirectory of working.
   chdir("/working/test");
-  getcwd(cwd, sizeof(cwd));
+  printf("getcwd() result: %s\n", getcwd(cwd, sizeof(cwd)));
   printf("Current working dir: %s\n", cwd);
 
   // Try to switch to a non-existent relative path from subdirectory.
   // Changing to working in /working/test should fail.
   chdir("working");
   printf("Errno: %s\n", strerror(errno));
   assert(errno == ENOENT);
-  getcwd(cwd, sizeof(cwd));
+  printf("getcwd() result: %s\n", getcwd(cwd, sizeof(cwd)));
```

Comment:
One reason for my feedback: I find it harder to read tests when the call being tested is itself hidden inside a parameter to something like printf (same goes for assert actually but I will admit I often do that one).